ERCS/erc-7629.md

```diff
@@ -0,0 +1,321 @@
+---
+eip: 7629
+title: Unified Token
+description: Unifying ERC-721 and ERC-20 for seamless asset conversion.
+author: 0xZeus1111 (@0xZeus1111), Nvuwa (@Nvuwa)
+discussions-to: https://ethereum-magicians.org/t/erc-7629-unified-token/18793
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-02-18
+requires: 20, 721
```

Comment:
As IERC7629 is going to create a new interface, better based on IERC165.

Reply:
Thank you for your advice. I will add the IERC165 interface to IERC7629.